src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Delete } from '@edx/paragon/icons';
+import { IconButton, Icon } from '@edx/paragon';
+import { connect } from 'react-redux';
+import ContentHighlightCardItem from '../ContentHighlightCardItem';
+import { useContentHighlightsContext } from '../data/hooks';
+import { generateAboutPageUrl } from '../data/constants';
+
+const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
+  const { deleteSelectedRowId } = useContentHighlightsContext();
+  const [deleteKey, setDeleteKey] = useState(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const {
+    title,
+    contentType,
+    partners,
+    cardImageUrl,
+    originalImageUrl,
+    firstEnrollablePaidSeatPrice,
+    aggregationKey,
+    key,
+  } = original;
+  // eslint-disable-next-line no-shadow
+  useEffect(() => {
+    if (deleteKey === aggregationKey) {
+      deleteSelectedRowId(deleteKey);
+      setIsDeleted(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [deleteKey, aggregationKey]);
+  if (isDeleted) {
+    return (
+      null
+    );
+  }
+  return (
+    <React.Fragment key={key}>
```

Comment:
I'd probably recommend rendering a `div` vs. a `React.Fragment` here so there is some DOM element grouping the 2 together (they are related elements, so should be grouped in some way).

Reply:
Using a div causes the items to render as the image below shows. I could of left an empty react fragment, but then the no key error occurs for each child element. React.Fragment was my solution to preventing both those issues.

![Screen Shot 2022-12-16 at 4 20 54 PM](https://user-images.githubusercontent.com/82611798/208191014-b67c2f7e-f978-41a4-ac86-425294a33bc6.png)